src/infra/repos/postgresql/prisma/schema.prisma

```diff
@@ -36,13 +36,33 @@ model Pet {
 }
```

Comment:
What you think about "breedAlias" logic? Because breed register has same logic that species, has an "other" field to unregistered breeds.

Reply:
yeah, it's make sense for me, I'm doing breed register this way.

Reply:
Would this `breedAlias` be a field in the `Species` table?

Reply:
In Pet table.